app/views/listings/index.html.erb

```diff
@@ -16,22 +13,21 @@
       </div>
       <div>
         <%= form_tag("/listings", method: "get") do %>
-        <div class="form-group">
-          <%= label_tag(:city, "City") %>
-          <%= text_field_tag(:city, "City", :value => nil)%>
-          What type of place would you like to stay in?
-          Home Type
-          <%= select_tag "category_id", options_for_select(categories_helper),:include_blank => true %>
-        </div>
-        <div class="form-group">
-          Nightly Rates
-          <%= select_tag "nightly_rate", options_for_select(nightly_rate_helper),:include_blank => true %>
-        </div>
-        <div class="form-group">
-          Guests
-          <%= select_tag "max_guests", options_for_select(guests_helper), :include_blank => true %>
-          <%= submit_tag "Filter", class: "btn btn-lg btn-default" %>
-        </div>
+          <div class="form-group">
+            <%= label_tag(:city, "City") %>
+            <%= text_field_tag(:city, "City", :value => nil)%>
+            Home Type
+            <%= select_tag "category_id", options_for_select(categories_helper),:include_blank => true %>
+          </div>
+          <div class="form-group">
+            Nightly Rates
+            <%= select_tag "nightly_rate", options_for_select(nightly_rate_helper),:include_blank => true %>
+          </div>
+          <div class="form-group">
```

Comment:
Good! We should def look into setting a default instead of showing the first option selections.